src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java

```diff
@@ -39,6 +39,10 @@ public class DeleghePGPagoPATest {
     private LoginPGPagoPATest loginPGPagoPaTest = new LoginPGPagoPATest();
 
 
+    public void vaiAllaSezioneDelegatiDellImpresa(){
+        logger.info("navigate to delegati");
+        driver.navigate().to("https://imprese." + System.getProperty("environment") + ".notifichedigitali.it/deleghe/delegati");
+    }
```

Comment:
The **button element** has already been implemented in some scenario use it instead of **driver methods**